subsys/net/lib/lwm2m/Kconfig

```diff
@@ -56,6 +56,16 @@ config LWM2M_ENGINE_MESSAGE_HEADER_SIZE
 	help
 	  Extra room allocated to handle CoAP header data
 
+config LWM2M_ENGINE_VALIDATION_BUFFER_SIZE
```

Comment:
As mentioned in the top-level comment, suggestion is to use CONFIG_LWM2M_ENGINE_VALIDATION_BUFFER_SIZE == 0 as "validation support is disabled".

Reply:
Makes sense, I'll implment it like this.